client/containers/DashboardActivity/DashboardActivity.tsx

```diff
@@ -43,7 +46,30 @@ const DashboardActivity = (props: Props) => {
 	});
 
 	return (
-		<DashboardFrame className="dashboard-activity-container" title="Activity">
+		<DashboardFrame
+			className="dashboard-activity-container"
+			title="Activity"
+			controls={
+				canEnrollInActivityDigestEmails && (
+					<Switch
+						checked={member.receivesDigestEmail}
+						className="parent-approval-switch"
+						onChange={(e) =>
+							api
+								.updateMember({
+									member,
+									update: {
+										receivesDigestEmail: (e.target as HTMLInputElement).checked,
+									},
+									scopeIds: { communityId: scope.communityId },
+								})
+								.then(setMember)
+						}
+						label="Enroll in Activity Digest Emails"
```

Comment:
$0.02 — I'd use "subscribe" over "enroll" here

Reply:
So "subscribe to"?

Reply:
I think so. Enrolling makes it sound...arduous...to me 😄